src/index.html

```diff
@@ -12,6 +12,161 @@
   </head>
   <body>
     <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="POST"
+      onsubmit="onSubmit()"
+    >
+
+      <fieldset class="block">
+        <legend>Personal information</legend>
+
+        <label for="surname" class="field">
+          Surname:
+          <input
+            id="surname"
+            name="surname"
+            type="text"
+            autocomplete="off">
```

Comment:
```suggestion
          <input
            id="surname"
            name="surname"
            type="text"
            autocomplete="off"
          >
```
Открывающую и закрывающую скобки тега располагай на одной линии